### PR TITLE
feat: Add `Repo.getWorktreeRoot()`

### PR DESCRIPTION
**Pull request by slarse**
#8 

Adds a `Repo.getWorktreeRoot()` method, which just returns the worktree root of the Git repository.

### CODE REVIEW

File: src/git.ts [hunk 1]
```diff
@@ -51,6 +51,15 @@ export class Repo {
     return this.gitExec(['diff', '-U0']);
   }
 
+  /**
+   * Determine the worktree root of this repository.
+   *
+   * @returns The worktree root
```
Comment (algomaster99) [inline src/git.ts:57]: You can reword this to `Fulfills with the worktree root`. It would be more correct plus you have done a similar thing for other methods in this class.

**Review by algomaster99 — COMMENTED** [review]
Just had one comment about the docstring. Rest looks good to me!

File: src/git.ts (hunk 1 above)
Comment (slarse) [reply src/git.ts:57]: Right, nice catch!
